Common/Interfaces/IAlgorithm.cs

```diff
@@ -450,6 +450,13 @@ TradeBuilder TradeBuilder
         Security AddSecurity(SecurityType securityType, string symbol, Resolution resolution, string market, bool fillDataForward, decimal leverage, bool extendedMarketHours);
 
         /// <summary>
+        /// Removes the security with the specified symbol. This will cancel all
+        /// open orders and then liquidate any existing holdings
+        /// </summary>
+        /// <param name="symbol">The symbol of the security to be removed</param>
+        bool RemoveSecurity(Symbol symbol);
```

Comment:
This should have been here since the `public QCAlgorithm.RemoveSecurity(Symbol)` method was added.
RemoveSecurity isn't used internally up to now, but is currently used in `AddRemoveSecurityRegressionAlgorithm`.